crawler.py

```diff
@@ -104,7 +113,7 @@
             site = "http://" + site
         manager.logger.info("Visiting %s..." % site)
         command_sequence = CommandSequence.CommandSequence(
-            site, reset=True
+            site, blocking=True, reset=True
```

Comment:
This looks important :)

Reply:
Yes, very much so :)

Unfortunately this means that there's no reason to run with more than one browser, since the TaskManager process now blocks on every command submission (even if there are other browsers available). This is a general issue of effectively having two layers of worker management. Long-term we should think about removing a bunch of the worker management logic from OpenWPM and instead rely on external tools to achieve parallelism (like we're doing with kubernetes).

Reply:
>  Long-term we should think about removing a bunch of the worker management logic from OpenWPM and instead rely on external tools to achieve parallelism (like we're doing with kubernetes).

Agree. Added tracking issue: https://github.com/mozilla/OpenWPM/issues/470